### ISSUE TITLE
Print Logic SQL / Actual SQL based on configuration

### ISSUE THREAD

**Issue by rahul-gupta-1525** [apache/shardingsphere#29913]
## Feature Request

Can we have configuration to print actual sql and logic sql in normal mode if configured? 

**sql-show.actual-sql**=true
**sql-show.logic-sql**=true

Today in normal mode ( **sql-show**=true ) both logic and actual sql logs are being printed. Actual sql query might be needed in logs for someone and logic sql query might be needed for someone and sometimes people would need both. Logic SQL would only be needed only during initial configuration and not needed as always

If we have a configuration, people might print based on their need

**Comment by LiberteKoder** [comment]
i configured like this   ` Properties otherProperties = new Properties();  otherProperties.setProperty("sql-show", "true");`  both types of SQL are printed,The current version should not support custom configurations

**Comment by rahul-gupta-1525** [comment]
> i configured like this ` Properties otherProperties = new Properties(); otherProperties.setProperty("sql-show", "true");` both types of SQL are printed,The current version should not support custom configurations

Yeah currently there is no custom configuration for printing granularly, it will be useful if we provide this configuration
